docs/guides/cache/index.md

```diff
@@ -1,87 +1,179 @@
 # Cache storage backends
 
-To ensure that builds run as quickly as possible, BuildKit automatically caches
-the build result in its own internal cache. However, in addition to this simple
-cache, BuildKit also supports exporting cache for a specific build to an
-external location to easily import into future builds.
+To ensure fast builds, BuildKit automatically caches the build result in its own
+internal cache. Additionally, BuildKit also supports exporting build cache to an
+external location, making it possible to import in future builds.
 
-This external cache becomes almost essential in CI/CD environments, where there
-may be little-to-no persistence between runs, but it's still important to keep
-the runtime of image builds as low as possible.
+An external cache becomes almost essential in CI/CD build environments. Such
+environments usually have little-to-no persistence between runs, but it's still
+important to keep the runtime of image builds as low as possible.
 
 > **Warning**
 >
-> If you use secrets or credentials inside your build process, then ensure you
-> manipulate them using the dedicated [--secret](../../reference/buildx_build.md#secret)
-> functionality instead of using manually `COPY`d files or build `ARG`s. Using
-> manually managed secrets like this with exported cache could lead to an
-> information leak.
+> If you use secrets or credentials inside your build process, ensure you
+> manipulate them using the dedicated
+> [--secret](../../reference/buildx_build.md#secret) functionality instead of
+> using manually `COPY`d files or build `ARG`s. Using manually managed secrets
+> like this with exported cache could lead to an information leak. {: .warning }
 
-Currently, Buildx supports the following cache storage backends:
+## Backends
 
-- `inline` image cache, that embeds the build cache into the image, and is pushed to
-  the same location as the main output result - note that this only works for the
-  `image` exporter.
+Buildx supports the following cache storage backends:
 
-  ([guide](./inline.md))
+- [Inline cache](./inline.md) that embeds the build cache into the image.
```

Comment:
I like this!

At some point I imagine us linking to a location in the reference docs when we have some more details to link to there, like in: https://github.com/docker/buildx/blob/master/docs/guides/drivers/index.md. Where would that link like that go in here, to keep consistency?

(note to self: if we change this here, we should change it everywhere in a follow-up)

Reply:
oh I see, yeah with multiple link targets it makes sense to stack them somehow. It just looked a bit odd/lonely with that one link to guides 😅 

like literally every technical writer ever, I have a thing for bullet lists, so maybe we could do something along these lines: (let's figure it out later when adding reference?)

<img width="1248" alt="image" src="https://user-images.githubusercontent.com/35727626/191586467-7d32c8cd-462a-4847-aa7d-ff524c581c37.png">

Reply:
Yup, that sounds good, once we have exporter + cache backends + driver docs I think we can do a final "consistency pass".